multicluster/cmd/multicluster-controller/member.go

```diff
@@ -63,6 +63,13 @@ func runMember(o *Options) error {
 			Client:    mgr.GetClient(),
 			namespace: env.GetPodNamespace()}})
 
+	hookServer.Register("/validate-multicluster-crd-antrea-io-v1alpha2-clusterset",
```

Comment:
Question - we also need to register the handler for a member cluster?

Reply:
Yes, this ClusterSet webhook was registered in controller.go before because there is no difference for the creation/update validation webhook in both leader and member clusters. But now, the deletion webhook is only required for the leader, so I moved this to the leader.go and member.go to get the `role` info. For the member cluster, it's no-ops for deletion event.